.gitignore

```diff
@@ -1 +1,5 @@
+*.swp
```

Comment:
May want to write `.*.swp` to be more precise, and maybe add `.*.swo`. But that's nitpicking.